packages/rest/test/routes.test.ts

```diff
@@ -293,20 +293,6 @@ describe("rest: routes", () => {
       strictEqual(diagnostics[0].message, "Param multiParam has multiple types: [query, path]");
     });
 
-    it("emit diagnostic when there are multiple unannotated parameters", async () => {
```

Comment:
replace this test with the behavior where all parameters become the body?

Reply:
Yeah, I meant to do that. Oops.